ux.symfony.com/src/Controller/LiveComponentDemoController.php

```diff
@@ -22,6 +22,7 @@ public function demoAutoValidatingForm(LiveDemoRepository $liveDemoRepository):
     {
         return $this->render('live_component_demo/auto_validating_form.html.twig', [
             'demo' => $liveDemoRepository->find('auto-validating-form'),
+            'demos' => $liveDemoRepository->findAll(),
```

Comment:
Is this a purposeful addition?

Reply:
I may or may not have tried some things with all the "demo" pages.... 👼 

Fixed